interface/src/app/gene/network/network.tpl.html

```diff
@@ -0,0 +1,15 @@
+<h1>ADAGE: Gene-Gene Network</h1>
+
+<div ng-hide="ctrl.statusMessage">
```

Comment:
As for the question on **ng-if**, it would be handy if Angular had both ""ng-if** and **ng-else**, but the latter is not supported, and the gene network should be shown only when **statusMessage** is a blank string. (Of course I can use two **ng-if**s to achieve the same effect.)